libtsuba/include/tsuba/RDGSlice.h

```diff
@@ -119,16 +130,13 @@ class KATANA_EXPORT RDGSlice {
 
   std::unique_ptr<RDGCore> core_;
   // NB: we intentionally do not include a property cache here because that will
-  // complicate the property cache logic; one could be added in the future if it
-  // seems necessary
-
-  void InitArrowVectors();
```

Comment:
Its not completely clear to me why we no longer would need to `InitArrowVectors()`? Does RDGCore now handle this?

Reply:
Yes it does